observability/power_monitoring/visualizing-power-monitoring-metrics.adoc

```diff
@@ -12,12 +12,13 @@ include::snippets/technology-preview.adoc[leveloffset=+2]
 You can visualize {PM-shortname} metrics in the {product-title} web console by accessing {PM-shortname} dashboards or by exploring *Metrics* under the *Observe* tab.
 
 include::modules/power-monitoring-dashboards-overview.adoc[leveloffset=+1]
-include::modules/power-monitoring-accessing-dashboards.adoc[leveloffset=+1]
+include::modules/power-monitoring-accessing-dashboards-admin.adoc[leveloffset=+1]
+include::modules/power-monitoring-accessing-dashboards-developer.adoc[leveloffset=+1]
 include::modules/power-monitoring-metrics-overview.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 [id="additional-resources_visualizing-power-monitoring-metrics"]
 == Additional resources
 * xref:../../observability/monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects_enabling-monitoring-for-user-defined-projects[Enabling monitoring for user-defined projects]
 
-//after the file is added to the observability directory, update xref path to ../observability/monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects_enabling-monitoring-for-user-defined-projects
+//after the file is added to the observability directory, update xref path to ../observability/monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects_enabling-monitoring-for-user-defined-projects
```

Comment:
Is this existing comment needed?

Reply:
Not really. Will remove this! Thanks! :)